sandworm/bin/config.ml

```diff
@@ -0,0 +1,6 @@
+let s3_bucket_ref = "provider:dune-binary-distribution"
+let s3_public_url = "WAIT FOR CLOUDFLARE ACCOUNT"
```

Comment:
I guess this needs to be filled out.

Reply:
Yes, otherwise the actions would fails every day...

Reply:
It will be `download.dune.ci.dev` in the end (Note to self)